server/routes/analyticsRouter.ts

```diff
@@ -185,6 +211,20 @@ export default function routes(router: Router): Router {
 
     const protectedCharacteristic = protectedCharacteristicRoutes[characteristicName].characteristic
 
+    const groupsForCharacteristic =
+      protectedCharacteristic === ProtectedCharacteristic.Age
+        ? knownGroupsFor(protectedCharacteristic).filter(ageGroup => ageGroup !== AgeYoungPeople)
+        : knownGroupsFor(protectedCharacteristic)
+
+    const trendsEntriesGroup = (req.query.trendsEntriesGroup || groupsForCharacteristic[0]) as string
```

Comment:
should probably 404 if `trendsEntriesGroup` is provided but not a known group? otherwise we could get inconsistencies